base/bucket_gocb.go

```diff
@@ -952,8 +896,85 @@ func (bucket CouchbaseBucketGoCB) Update(k string, exp int, callback sgbucket.Up
 }
 
 func (bucket CouchbaseBucketGoCB) WriteUpdate(k string, exp int, callback sgbucket.WriteUpdateFunc) error {
-	LogPanic("Unimplemented method: WriteUpdate()")
-	return nil
+
+	bucket.singleOps <- struct{}{}
+	defer func() {
+		<-bucket.singleOps
+	}()
+
+	// The number of Couchbase Nodes that durable writes should be replicated to or persisted to
```

Comment:
Can you add a comment with the details on what replicateTo=0 and persistTo=1 mean, so that it's there in 6 months when we're looking at this again?